### ISSUE TITLE
Add test for climatic_summary that contains a `summaries_params` parameter

### ISSUE THREAD

**Issue by lloyddewit** [IDEMSInternational/cdms.products#81]
`climatic_summary()` has a `summaries_params` parameter.
This parameter has one list item for each summary function, and one list of parameters for  each summary function e.g. 'list(mean = list(trim = 0.5))'.
In the Python wrapper, this parameter is translated from Python to R using some special code. I would like to test this translation.
Please could we add some `climatic_summary()` tests that have meaningful values for `summaries_param` (i.e. values that affect the output)?

**Comment by lloyddewit** [comment]
@HawardKetoyoMsatsi Please could you look at this? Thanks
(I don't have rights in this repo to assign people)

**Comment by dannyparsons** [comment]
Good idea. @HawardKetoyoMsatsi Can you add a test which has the `summaries_param` argument included?

I think the example Stephen gave is sensible `list(mean = list(trim = 0.5))` because `trim` is a parameter of `mean` and it will change the result. 

Let me know if this makes sense or you have any questions.

**Comment by dannyparsons** [comment]
(You may need to change the value of `trim` to get a substantially different output.)
